apis/v0/jwt.go

```diff
@@ -102,3 +108,45 @@ func ClearJustifications(t jwt.Token) error {
 
 	return t.Remove(jwtJustificationsKey)
 }
+
+// ToJSON converts the token into json with justification claims seperated from other claims.
+func ToJSON(ctx context.Context, t jwt.Token) ([]byte, error) {
+	return marshal(ctx, t, "\n---", func(v any) ([]byte, error) { return json.MarshalIndent(v, "", "  ") })
```

Comment:
This is using MarshalIndent, so it's actually "pretty json". It also doesn't look like we return valid JSON, which could be confusing to callers. L149 appends `"\njustifications\n"` which would make the JSON and YAML invalid to parse.

Reply:
Ah..ok, I can only think of two ways: 1. convert the whole claimsMap to json without separating justs. 2. convert the whole claimsMap to json + create a different method to convert justs to json, so for validate cli command, it will call both funcs it will output justs twice one in the whole json + one separated. Do you have other suggestions?

Reply:
Is it acceptable to combine claims -> claimsMapNoJusts and justifications -> justs list to a new map, and marshal the new map to json/yaml?

Reply:
iirc, the original goal here was to pretty-print a JWT on a terminal. If we just want a jwt as JSON, we can call `json.Marshal` directly on the JWT and get _basically_ what you have here.